install/installing-with-kata-manager.md

```diff
@@ -0,0 +1,24 @@
+# Installing with kata-manager
+
+* [Introduction](#Introduction)
+* [Install the Kata packages](#install-the-kata-packages)
+* [Install Docker](#install-docker)
+
+## Introduction
+If your distribution is supported, you can use [`kata-manager`](https://github.com/kata-containers/tests/blob/master/cmd/kata-manager)
+to avoid copying or typing installation instructions by hand.
+
+## Install the Kata packages
+```
+bash -c "$(curl -fsSL \
+https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) \
+install-packages"
+```
+
+## Install Docker
```

Comment:
Might be worth mentioning that kubernetes support is not available with this tool. Also, you could mention that the user can run `kata-manager.sh -h` for full details.